src/config/index.js

```diff
@@ -74,6 +69,17 @@ const config = convict({
     ],
     default: process.env.ENVIRONMENT ?? 'local'
   },
+  // TODO - do we need the random hash at the end of the bucket name?
+  bucket: {
+    local: {
```

Comment:
we can manage the different buckets by binding it to an environment variable and setting it in the per-environment app-config